services/api-server/src/simcore_service_api_server/core/settings.py

```diff
@@ -1,6 +1,7 @@
+from collections.abc import Mapping
```

Comment:
Ruff is suggesting to import Mapping this way

Reply:
I recall thisone was because they will be moving Mapping, Callable and all these here and deprecate them from typing. Check the link provided by ruff when it complains for more info